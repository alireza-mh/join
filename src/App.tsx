import React from 'react'

const App = () => (
  <div>
    <p>Recruiter Dashboard</p>
  </div>
)

export default App
