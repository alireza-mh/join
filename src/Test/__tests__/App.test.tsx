import React from 'react'
import { render } from '@testing-library/react'

import App from '../../App'

test('renders', () => {
  const { container } = render(<App />)
  expect(container).toMatchSnapshot()
})
