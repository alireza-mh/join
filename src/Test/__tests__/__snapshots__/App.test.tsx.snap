// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div>
  <div
    class="join-layout"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg"
    >
      <div
        class="join-review-candidates"
      >
        <div
          class="join-review-candidates__loading"
        >
          <div
            class="MuiCircularProgress-root MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
            role="progressbar"
            style="width: 40px; height: 40px;"
          >
            <svg
              class="MuiCircularProgress-svg"
              viewBox="22 22 44 44"
            >
              <circle
                class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
                cx="44"
                cy="44"
                fill="none"
                r="20.2"
                stroke-width="3.6"
              />
            </svg>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
